### ISSUE TITLE
Doesn't work with multiple network interfaces

### ISSUE THREAD

**Issue by XavierBestelSonos** [jjakobh/ssdp-client#9]
Hi,

When running the example, with a certain number of network interfaces it works and shows me all my SSDP devices, but if I add an unused link-local interface, it doesn't find anything.  
IMHO it should probably bind to all available interfaces for it work work properly; right now it seems it binds to the first one available which may or may not work by luck.

**Comment by XavierBestelSonos** [comment]
More tests:
- I have 2 interfaces up, with devices on both: only devices on 1 interface show
- I have 3 interfaces up, 2 with devices and 1 without; no devices show

So it seems to confirm `ssdp-client` connects to one interface, whichever comes first.

**Comment by dheijl** [comment]
This is possibly my fault for suggesting a change in issue #3. 
I suppose that it would be better to simply iterate the SSDP discovery over all the active interfaces, or let the user select an interface for the discovery (this is what I eventually ended up with, I only use the code from issue #3 now to suggest a plausible default).

**Comment by bestouff** [comment]
Yes, both iterating all interfaces or being able to select one are sensible choices.

**Comment by jwalton** [comment]
I’m looking to be able to select a specific interface.  I’m my use case has a device with a public facing interface and a private facing interface, and I only want to try to discover devices on the private network.  Happy to submit a PR if you have a suggestion for how you want this done?  Could be as simple as just specifying the bind address as an option.
